src/locales/id/translation.json

```diff
@@ -0,0 +1,167 @@
+    "checking_files": "Memeriksa file {{title}}… ({{percentage}} selesai)",
+    "downloading": "Mengunduh {{title}}… ({{percentage}} selesai) - Perkiraan {{eta}} - {{speed}}"
+  },
+  "catalogue": {
+    "next_page": "Halaman berikutnya",
+    "previous_page": "Halaman sebelumnya"
+  },
+  "game_details": {
+    "open_download_options": "Buka opsi unduhan",
+    "download_options_zero": "Tidak ada opsi unduhan",
+    "download_options_one": "{{count}} opsi unduhan",
+    "download_options_other": "{{count}} opsi unduhan",
+    "updated_at": "Diperbarui {{updated_at}}",
+    "install": "Install",
+    "resume": "Lanjutkan",
+    "pause": "Hentikan sementara",
+    "cancel": "Batalkan",
+    "remove": "Hapus",
+    "remove_from_list": "Hapus",
+    "space_left_on_disk": "{{space}} tersisa pada disk",
+    "eta": "Perkiraan {{eta}}",
+    "downloading_metadata": "Mengunduh metadata…",
+    "checking_files": "Memeriksa file…",
+    "filter": "Saring repacks",
+    "requirements": "Keperluan sistem",
+    "minimum": "Minimum",
+    "recommended": "Rekomendasi",
+    "no_minimum_requirements": "{{title}} Tidak ada informasi kebutuhan sistem",
+    "no_recommended_requirements": "{{title}} Tidak ada informasi rekomendasi kebutuhan sistem",
+    "paused_progress": "{{progress}} (Terhenti)",
+    "release_date": "Dirilis pada {{date}}",
+    "publisher": "Dipublikasikan oleh {{publisher}}",
+    "copy_link_to_clipboard": "Copy tautan",
+    "copied_link_to_clipboard": "Tautan dicopy",
+    "hours": "jam",
+    "minutes": "menit",
+    "amount_hours": "{{amount}} jam",
+    "amount_minutes": "{{amount}} menit",
+    "accuracy": "{{accuracy}}% akurasi",
+    "add_to_library": "Tambahkan ke koleksi",
+    "remove_from_library": "Hapus dari koleksi",
+    "no_downloads": "Tidak ada unduhan tersedia",
+    "play_time": "Dimainkan selama {{amount}}",
+    "last_time_played": "Terakhir dimainkan {{period}}",
+    "not_played_yet": "Kamu belum memainkan {{title}}",
+    "next_suggestion": "Rekomendasi berikutnya",
+    "play": "Mainkan",
+    "deleting": "Menghapus installer…",
+    "close": "Tutup",
+    "playing_now": "Memainkan sekarang",
+    "change": "Ubah",
+    "repacks_modal_description": "Pilih repack yang kamu ingin unduh",
+    "downloads_path": "Lokasi Unduhan",
+    "select_folder_hint": "Untuk merubah folder bawaan, akses melalui",
+    "settings": "Pengaturan",
+    "download_now": "Unduh sekarang",
+    "installation_instructions": "Instruksi Instalasi",
+    "installation_instructions_description": "Langkah tambahan dibutuhkan untuk meng-instal game ini",
+    "online_fix_instruction": "OnlineFix games mebutuhkan kata sandi untuk ekstraksi. Saat diperlukan, gunakan kata sandi ini:",
+    "dodi_installation_instruction": "Saat menjalankan DODI installer, tekan tombol atas pada keyboard <0 /> untuk melanjutkan proses instalasi:",
+    "dont_show_it_again": "Jangan tunjukkan lagi",
+    "copy_to_clipboard": "Copy",
```

Comment:
Are these two left behind? Or they're really the same in Indonesian?

Reply:
actually some part of english terms are still used commonly, but hold on, i think i can translate that as well.